task/packrat-api/src/lib.rs

```diff
@@ -0,0 +1,37 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+//! Client API for the VPD task.
+
+#![no_std]
+
+use derive_idol_err::IdolError;
+use userlib::*;
+use zerocopy::{AsBytes, FromBytes, LittleEndian, U16};
+
+pub use oxide_barcode::VpdIdentity;
```

Comment:
Is re-exporting this necessary?

Reply:
Not strictly necessary, but I think it should be - `VpdIdentity` is part of this crate's public API (although it's hidden here because it's part of the generated idol client), and I don't like when crates require users to depend on _another_ crate for types they use.

Reply:
Seems reasonable!